packages/dashboard/src/components/Wizard/PanelWizard.styles.ts

```diff
@@ -0,0 +1,45 @@
+import { IPanelStyles, ITheme } from 'office-ui-fabric-react';
+import { IWizardStyles } from './Wizard.types';
+import { IPanelWizardStyles, IPanelWizardStyleProps } from './PanelWizard.types';
+
+export const defaultPanelStyleSet = (theme: ITheme): Partial<IPanelStyles> => {
+  return {
+    content: {
+      height: '100%',
+      padding: 0
+    },
+    commands: {
+      display: 'flex',
+      flexDirection: 'row',
+      borderBottom: `1px solid ${theme.semanticColors.bodyDivider}`
+    },
+    scrollableContent: {
+      overflowY: 'hidden'
```

Comment:
Assuming this is because the wizard has its own content with scrolling?

Reply:
Yes I expect each step to scroll within the content area, instead of the whole wizard scroll together.